pkgs/development/tools/misc/texinfo/common.nix

```diff
@@ -40,7 +40,7 @@ stdenv.mkDerivation {
 
   # ncurses is required to build `makedoc'
   # this feature is introduced by the ./cross-tools-flags.patch
-  NATIVE_TOOLS_CFLAGS = if crossBuildTools then "-I${getDev buildPackages.ncurses}/include" else null;
+  NATIVE_TOOLS_CFLAGS = lib.optionals crossBuildTools [ "-I${getDev buildPackages.ncurses}/include" ];
```

Comment:
Might as well drop this hunk now, right?

Reply:
Oh, oops. I didn't realize that's not how it started...